### PR TITLE
[14.0] [MIG] sale_order_partner_restrict

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Comment by GuillemCForgeFlow** [comment]
LGTM :+1: Just one minor thing, the commit message should be Migration to 14.0, right?

**Comment by JasminSForgeFlow** [comment]
> LGTM +1 Just one minor thing, the commit message should be Migration to 14.0, right?

Yes, right.
Its updated
Thanks

**Review by GuillemCForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Review by MiquelRForgeFlow — APPROVED** [review]

File: sale_order_partner_restrict/models/sale.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                "|",
+                ("parent_id", "=", False),
+                ("type", "=", "contact"),
+            ]
+        return partner_restrict_domain
+
+    @api.onchange("sale_order_partner_restrict")
+    def _onchange_sale_order_partner_restrict(self):
+        self.ensure_one()
+        return {"domain": {"partner_id": self._get_partner_restrict_domain()}}
```
Comment (rousseldenis) [inline sale_order_partner_restrict/models/sale.py:48]: This is deprecated, but still working. Just to pay attention

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**